packages/office-ui-fabric-react/src/components/SpinButton/docs/new/SpinButtonBestPractices.md

```diff
@@ -0,0 +1,12 @@
+### Layout
+
+- Use a SpinButton when you need to incrementally change a value.
+- Use a SpinButton when values are tied to a unit of measure.
+- Don’t use a SpinButton for binary settings.
+- Don't use a SpinButton for a range of three values or less.
+- Place labels to the left of the spinbutton control. For example, Length of ruler (cm).
```

Comment:
This guidance seems a bit odd. SpinButton is just another single-line form field, and for most controls like that, the label can be either on the left or on top.